go.mod

```diff
@@ -2,4 +2,8 @@ module github.com/trussworks/terraform-aws-ecs-service
 
 go 1.13
 
-require github.com/gruntwork-io/terratest v0.23.3
+require (
+	github.com/aws/aws-sdk-go v1.23.8
```

Comment:
AWS Go SDK is up to 1.28.2 -- dependabot will be  your friend :P

Reply:
thanks Barry